### DISCUSSION TITLE
Does Splink create its own spark context separate from the spark context it was ran in?

### DISCUSSION THREAD

**Discussion by jkginfinite** [moj-analytical-services/splink#1625] [Q&A]
Does Splink create it's own spark context separate from the spark context that is initiated when running on EMR et al spark platforms?

 

If so, how do we merge the Splink spark context with the original context?

**Comment by RobinL** [comment]
Splink retrieves the spark context from the Spark data frames you pass into it.

But you can explicitly pass Spark into the linker if you wish
https://github.com/moj-analytical-services/splink/blob/360a413eafcf5c17d03673c16e86ae1e1b68da6f/splink/spark/linker.py#L90
